VR/VRBase.h

```diff
@@ -0,0 +1,13 @@
+#ifndef __VR_BASE
+#define __VR_BASE
```

Comment:
Generally we use `#pragma once` instead of these ifdefs. It's universally supported since many years and do essentially the same thing.